deepchecks/vision/checks/performance/robustness_report.py

```diff
@@ -271,31 +299,6 @@ def _create_top_affected_graph(self, top_affected_dict):
         return fig
 
 
-def get_robustness_augmentations(dataset):
```

Comment:
I'm actually not a fan of having big if-albumentation elif imgaug across the code as it can go wrong fast. I'd like to this this abstracted away in the inialization function of a wrapping class. Initialize the augmentation classes there, self-assign class methods and from that point onwards simply calling them will be agnostic to the library used by the user.